Makefile

```diff
@@ -107,6 +109,10 @@ help: # Show this help
 	echo ''; \
 	} 1>&2; \
 
+clean: # Remove build directory and bindata-generated files
```

Comment:
Do we need `.PHONY` for the new targets?

Reply:
clean needs to be marked as PHONY, but the two bindata.go files aren't PHONY, so they aren't tagged as such. I'll update the PR.